server/controllers/docHandlers.js

```diff
@@ -76,7 +79,10 @@ const unlockDoc = (req, res) => {
   }
   else{
     Doc.findOne({_id:req.params.documentId}, function(lockErr, lockResult){
-      if(lockErr || (lockresult.locked != req.user._id && lockresult.owner_id != req.user._id)) {
+      if(lockErr){
+        return res.status(500).json({error:"Failed to check the lock on document"});
+      }
+      if(lockresult.locked != req.user._id && lockresult.owner_id != req.user._id) {
```

Comment:
What's the point of the second condition?

Reply:
in the details it says that owners are able to force unlock a shared document even if someone else has it

Reply:
Then it should be an OR condition

Reply:
nvm it's a negation